### ISSUE TITLE
Padronização de Estilo nas Páginas do Quizz

### ISSUE THREAD

**Issue by falvojr** [digitalinnovationone/roadmaps#40]
Esta melhoria tem como objetigo padronizar a página de Quizz (onde as 16 perguntas são apresentadas) para que tenham a mesma identidade visual da Home e Páginas de Roadmap. Além disso, talvez tenhamos que pensar em algumas melhorias de responsividade e mobile-first 😉
Periodicamente, nós da DIO, criaremos Issues como esta para formalizar algumas evoluções e melhorias que estão no nosso radar, mas toda comunidade é bem vinda para desenvolvê-las. É só responder aqui que planejamos o desenvolvimento/revisão dessas ocorrências.

Em tempo, aproveito para fazer um gradecimento especial à @elidianaandrade e @refusado pela contribução incrível na identidade visual e melhorias funcionais em nosso Roadmap 🤩

**Comment by elidianaandrade** [comment]
Obrigada! Eu havia alterado a parte das cores inserindo as variáveis root, mas não commitei, pois estava pensando no design. 
Uma das coisas que pensei é deixar mais expresso o que cada número significa. Seguindo a Likert scale, o 3 seria "neutro", mas acredito que sem está expresso cada pessoa vai ter uma percepção diferente do que seria a escala 3.

**Comment by falvojr** [comment]
Perfeito @elidianaandrade, muito obrigado pela proatividade 👊🤩
Concordo que poderíamos deixar mais claro o significado da escala Likert, talvez incluir uma legenda dizendo o que cada valor representa, por exemplo (de 1 a 5): ` Discordo Totalmente / Discordo / Neutro / Concordo / Concordo Totalmente`. Em termos visuais, poderíamos colocar um hint ou escrever abaixo da escala, por exemplo. O que acha que seria mais interessante?

Uma parada que percebi no Mobile, que talvez tenhamos que ajustar é o tamanho da fonte da pergunta, que está bem pequena, talvez tenhamos que revisar isso também 😊

**Comment by falvojr** [comment]
Concordo demais @refusado, são evoluções bem interessantes mesmo! Podemos ir alinhando tecnicamente as melhorias por aqui e cada um pode ir contribuindo com algo. Acredito que a @elidianaandrade já tem algumas melhorias na manga, principalmente a nível de UI (e também UX em alguns aspectos)... Em termos de checklist pensei no seguinte:

- [ ] **Redesign da Tela de Quizz** (@elidianaandrade)
- [ ] **Informações/Legenda na Escala Likert** (1 a 5): caso seja necessário descrições específicas da escala por pergunta (por exemplo, uma com `Discordo Totalmente / Discordo / Neutro / Concordo / Concordo Totalmente` e outra com `Não é Importante / Pouco Importante / Moderado / Importante / Muito Importante`) podemos incluir essas informções no JSON que tem as perguntas;
- [ ] **Melhorar Experiência em Dispositivos Móveis (Mobile First)**
- [ ] **Melhorar Dinamismo na Troca de Perguntas**
- [ ] **Criar Barra de Progresso**

O que acham?

**Comment by elidianaandrade** [comment]
Curti! Essas alterações vão tornar a navegação ainda mais interessante. Sobre a legenda, penso que podemos analisar as perguntas e adotar uma que seja compatível com todas as elas, para manter um padrão na interface e ser automático para a pessoa que está lendo.
Também gostei das sugestões sobre animações e a barra de progresso.

**Comment by refusado** [comment]
Achei ótimo!

(Estava digitando aqui e a @elidianaandrade já disse o que eu iria propor kkkk vou mandar mesmo assim)
Em relação às legendas da Escala Likert, acredito que podemos facilitar ainda mais:
Que tal, ao invés de criar "respostas" específicas nas perguntas, nós adaptarmos as perguntas para uma só escala.

Por exemplo: `Você se considera uma pessoa criativa?` -> `O quanto você se considera uma pessoa criativa?`

**Comment by elidianaandrade** [comment]
Isso mesmo, @refusado! É bom, pois a pessoa se concentra em ler somente a pergunta.

**Comment by refusado** [comment]
Exato!! Acho que é bem melhor para o usuário, é menos _trabalho_ para ele

**Comment by falvojr** [comment]
Perfeito pessoal, concordo totalmente 🤩
Vou revisar as perguntas então e deixá-las padronizadas e aderentes a um mesmo padrão. Assim que possível conpartilho com vocês aqui 👊😊

**Comment by falvojr** [comment]
Fala @elidianaandrade e @refusado, apenas atualizando vocês. Troquei uma ideia com o @diegobrunoDIO e ele vai nos ajudar a padronizar as perguntas. Em breve ajustamos e já podemos seguir com os itens seguintes do nosso checklist 🥰

**Comment by elidianaandrade** [comment]
Gostei das legendas, só tenho uma sugestão a respeito do direcionamento das perguntas para o nível de interesse da pessoa, pois penso que ela pode responder no sentido geral e considerar tudo importante 🤔

As possibilidades que pensei foram:
1. "**Para você, qual o nível de importância / quão importante** é o design no desenvolvimento de um projeto?";
2. Alterar "importante" para "interessante";
3. Manter as perguntas como estão e na página inicial instruir que a pessoa deve responder com base no nível de importância que ela considera.

O que acham?

**Comment by elidianaandrade** [comment]
Outra coisa que pensei, mas deixei para sugerir depois que tivesse as legendas, é na versão mobile colocar os botões de 1 a 5 um por linha (com as legendas ao lado), para que preencha toda a tela.

**Comment by falvojr** [comment]
Perfeito @elidianaandrade, concordo com ambos comentários. Sobre o primeiro, acredito que seria interessante trocarmos para interessante mesmo, assim mantemos o foco nas preferências dos usuários e não em algo mais subjetivo como a importância. Pensei no seguinte, o que acham?

1. Qual o seu nível de interesse na **prototipação** (design/desenho inicial) de um software?
2. Qual o seu nível de interesse em atividades que favorecem a **criatividade** no desenvolvimento de um software?
3. Qual o seu nível de interesse na **interface (UI) e experiência (UX)** proporcionadas em um software?
4. Qual o seu nível de interesse em aspectos de **performance** em um software?
5. Qual o seu nível de interesse no desenvolvimento do **raciocínio lógico** como uma ferramenta para atuar na área de computação?
6. Qual o seu nível de interesse em **matemática e estatística** como ferramentas para atuar na área de computação?
7. Qual o seu nível de interesse em **Inteligência Artificial (IA)**?
8. Qual o seu nível de interesse no desenvolvimento de **aplicativos para celulares**?
9. Qual o seu nível de interesse em **segurança da informação**?
10. Qual o seu nível de interesse em **computação em nuvem**?
11. Qual o seu nível de interesse na **análise de grandes volumes de dados** para tomadas de decisão?
12. Qual o seu nível de interesse na **avaliação da qualidade** como fator determinante no sucesso de projetos?
13. Qual o seu nível de interesse no desenvolvimento de soluções de Internet para **casas inteligentes, indústrias e carros autônomos** na área de computação?
14. Qual o seu nível de interesse em habilidades de **liderança** e **trabalho em equipe** no contexto de gestão de projetos?
15. Qual o seu nível de interesse na tomada de **decisões gerenciais** no contexto de gestão de projetos?
16. Qual o seu nível de interesse em tecnologias para o desenvolvimento de **jogos**?

**Comment by falvojr** [comment]
Talvez possamnos até pensar em algo que dê menos ênfase visual a esse início de frase que se repete "Qual o seu nível de interesse...". Enfim, só uma ideia

**Comment by refusado** [comment]
Curti muito as perguntas e gostei também das sugestões, vocês arrasaram demais!

Para os botões em linha eu acho que ficaria bom porém acho que seria interessante tentar manter todo o conteúdo da página dentro da viewport - sem que exceda a tela do dispositivo, pois acredito que se for necessário "scrollar" a página em cada pergunta para selecionar os botões mais em baixo pode ser desfavorável à experiência do usuário.

**Comment by falvojr** [comment]
@refusado conseguiria assumir a atualização das mensagens? Com isso, acredito que finalizamos o checklist que propomos nesta melhoria:

- [ ] **Padronização das Perguntas**
- [x] **Redesign da Tela de Quizz**
- [ ] **Informações/Legenda na Escala Likert**
- [x] **Melhorar Experiência em Dispositivos Móveis (Mobile First)**
- [x] **Melhorar Dinamismo na Troca de Perguntas**
- [x] **Criar Barra de Progresso**

**Comment by elidianaandrade** [comment]
Show! @refusado isso mesmo, como são apenas 5 tem como manter sem "scrollar".

**Comment by refusado** [comment]
Posso atualizar trabalhar na atualização das mensagens sim.
Acabei de ver as alterações que fizeram até agora e curti demais, a página do quizz ficou realmente muito melhor. Vocês arrasam!

**Comment by refusado** [comment]
@elidianaandrade, você que está mais por dentro das melhorias na interface, não acha que centralizar verticalmente a pergunta + botões pode ser uma opção? Os elementos da tela estão todos no topo e, principalmente para o mobile, há um grande espaço em branco no meio da tela. Acho que essa distribuição pode deixar um pouco melhor, talvez.

<img width="25%" align="left" src="https://user-images.githubusercontent.com/89546855/223908845-467b13c9-e4dd-4952-ae21-ecf5cf3324e6.png">
<img width="25%" src="https://user-images.githubusercontent.com/89546855/223908213-011968ba-1d2d-483c-8bbb-5441b1b28b8e.png">

**Comment by elidianaandrade** [comment]
@refusado essa versão ficaria melhor mesmo, pois realmente ficou esse espaço, que como disse seria preenchido com as legendas na vertical depois da atualização das perguntas.

 Eu tinha ficado na dúvida inicialmente se centralizava ou não, pois havia testado no desktop centralizado, mas achei a navegação muito "movimentada", pois os elementos ficavam subindo e descendo a depender do tamanho da pergunta, teria como solucionar isso definindo a altura das perguntas de modo que isso não ocorresse, só que o espaçamento entre elas e os outros elementos iria modificar. Como no desktop não tinha ficado tanto espaço como no mobile, acabei mantendo no topo, mas não é como se tivesse "batido o martelo" sobre isso.

**Comment by falvojr** [comment]
@elidianaandrade e @refusado muiuto obrigado pelo apoio incrível nesta Issue! Considerando o ultimos PRs, veem algo pendente por aqui?

**Comment by elidianaandrade** [comment]
Obrigada! Seguindo o checklist, acredito que só falta a legenda agora.

**Comment by refusado** [comment]
Achei que eu tinha respondido isto quando fui mencionado, estava olhando agora o projeto e vi que ficou faltando minha mensagem. Me desculpem :c. @falvojr já pode esta issue!

Ficou perfeito o projeto, voces são feras.

**Comment by falvojr** [comment]
Vlw demais @elidianaandrade, @refusado e @diegobrunoDIO, vocês são incríveis 🤩
